pkg/loadbalance/loadbalancer.go

```diff
@@ -0,0 +1,34 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package loadbalance
+
+// LoadBalance - general interface explaining the API of all load balancers available in the package
+type LoadBalancer interface {
```

Comment:
Avoids "stutter" as loadbalance is the package name.

```suggestion
type Interface interface {
```